app/views/data_lists/_form.html.erb

```diff
@@ -23,17 +23,23 @@
     <%= f.label :startyear %><br />
     <%= f.text_field :startyear %>
   </div>
+  <% if request.path =~ /edit/ %>
+    <div class="field">
+      <%= f.label :owned_by %><br />
+      <%= f.select(:owned_by, User.all.map {|u| [u.email, u.id]}, :prompt => "Owned by") %>
+    </div>
+  <% end %>
   <!-- <div class="field">
-    <%= f.label :endyear %><br />
-    <%= f.text_field :endyear %>
+    < %= f.label :endyear %><br />
```

Comment:
Let's get in the habit of deleting commented code, unless it is accompanied by sufficient documentation to validate its continued existence.